src/main/webapp/homepage.jsp

```diff
@@ -42,29 +42,6 @@
     </div>
 
     <div class="bg-gray-100 py-8">
-<!-- 	    <input  -->
-<!-- 					    id="searchbar"  -->
-<!-- 					    onkeyup="search_course()"  -->
-<!-- 					    type="text"  -->
-<!-- 					    name="search"  -->
-<!-- 					    placeholder="Search Courses"  -->
-<!-- 					    class=" -->
-<!-- 					    	block -->
-<!-- 					        mb-4 -->
-<!-- 					        px-4 -->
-<!-- 					        py-2 -->
-<!-- 					        w-1/4 -->
-<!-- 					        border  -->
-<!-- 					        border-gray-300  -->
-<!-- 					        rounded-lg  -->
-<!-- 					        focus:outline-none  -->
-<!-- 					        focus:ring-2  -->
-<!-- 					        focus:ring-blue-500  -->
-<!-- 					        focus:border-blue-500  -->
-<!-- 					        text-gray-700  -->
-<!-- 					        placeholder-gray-400  -->
-<!-- 					        shadow-sm -->
-<!-- 					    "> -->
```

Comment:
are these lines supposed to be deleted?

Reply:
Yeah, they are just a comment

Reply:
oh my bad I deleted this part accidently

Reply:
alright then that's fine